test/puppetlabs/puppetdb/testutils.clj

```diff
@@ -23,45 +23,36 @@
          :prefix "/pdb/cmd"
          :version :v1))
 
-(defn test-db-config
-  "This is a placeholder function; it is supposed to return a map containing
-  the database configuration settings to use during testing.  We expect for
-  it to be overridden by another definition from the test config file, so
-  this implementation simply throws an exception that would indicate that our
-  config file was invalid or not read properly."
+(defn create-hsqldb-map
+  "Returns a database connection map with a reference to a new in memory HyperSQL database"
   []
-  (throw (IllegalStateException.
-          (str "No test database configuration found!  Please make sure that "
-               "your test config file defines a no-arg function named "
-               "'test-db-config'."))))
-
-(defn load-test-config
-  "Loads the test configuration file from the classpath.  First looks for
-  `config/local.clj`, and if that is not found, falls back to
-  `config/default.clj`.
-
-  Returns a map containing the test configuration.  Current keys include:
-
-    :testdb-config-fn : a no-arg function that returns a hash of database
-        settings, suitable for passing to the various `clojure.java.jdbc`
-        functions."
+  {:classname "org.hsqldb.jdbcDriver"
+   :subprotocol "hsqldb"
+   :subname (str "mem:"
+                 (java.util.UUID/randomUUID)
+                 ";hsqldb.tx=mvcc;sql.syntax_pgs=true")})
+
+(defn get-env [var & [default]]
+  (or (System/getenv var) default))
+
+(def create-postgres-map
```

Comment:
this is a `def`, but the name indicates it's a function. `create-hsqldb-map` is a function, presumably because it needs to generate a uuid. I think this one should be a function as well for consistency.